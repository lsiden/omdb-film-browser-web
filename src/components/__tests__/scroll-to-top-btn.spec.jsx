import React from 'react'
import { create } from 'react-test-renderer'

import ScrollToTopButton from 'components/scroll-to-top-btn'

const createWrapper = () => create(<ScrollToTopButton />)

describe('ScrollToTopButton', () => {
  it('matches snapshot', () => {
    expect(createWrapper).toMatchSnapshot()
  })
})
