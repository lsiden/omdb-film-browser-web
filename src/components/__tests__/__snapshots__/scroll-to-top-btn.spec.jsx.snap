// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTopButton matches snapshot 1`] = `[Function]`;
